tests/datasets/test_transaction_processor.py

```diff
@@ -49,11 +52,13 @@ def serialize(self) -> Tuple[int, str, Mapping[str, Any]]:
                 "event_id": self.event_id,
                 "message": "/organizations/:orgId/issues/",
                 "group_id": None,
+                "group_ids": self.group_ids,
```

Comment:
Ideally, it's best to have `group_ids` only in 1 part of the message, not both. The message sizes get bigger unnecessarily and there is really no value.